tests/cli_legacy.py

```diff
@@ -334,6 +334,13 @@ def test_static_ipv4_netmask(self):
             'version': 2,
             'ethernets': {'en1': {'addresses': ["1.2.3.4/8"]}}}}, out.decode())
 
+    def test_static_ipv4_alias(self):
+        out = self.do_test('auto en1\niface en1 inet static\naddress 1.2.3.4/8\n'
+                           'auto en1:1\niface en1:1 inet static\naddress 1.2.3.5/8', dry_run=True)[0]
+        self.assertEqual(_load_yaml(out), {'network': {
+            'version': 2,
+            'ethernets': {'en1': {'addresses': ["1.2.3.4/8", "1.2.3.5/8"]}}}}, out.decode())
```

Comment:
thought (non-blocking): I'm pondering if this should be translated to something like that instead:

```
version: 2
ethernets:
  en1:
    addresses:
      - "1.2.3.4/8"
      - "1.2.3.5/8":
          label: "en1:1"
```

But this could also be done as a separate PR, as currently this is a targeted fix for a specific problem. The solution could be enhanced to retain more information via the `label`.